.circleci/config.yml

```diff
@@ -114,7 +114,7 @@ install_dep_190: &install_dep_190
         # check if we have restored venv cache (/home/circleci/venv) correctly, if so, just skip
         if [ -f /home/circleci/venv/check_version.py ]; then python /home/circleci/venv/check_version.py torch eq 1.8 && exit 0; fi
         # start installing
-        pip install --pre --progress-bar off torch==1.9.0.dev20210330+cu101 torchvision==0.10.0.dev20210330+cu101 -f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html
+        pip install --pre --progress-bar off torch==1.9.0.dev20210415+cu101 torchvision==0.10.0.dev20210415+cu101 -f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html
```

Comment:
this is strange. it shouldn't show up in your PR. can you make sure you merge the master, not cherry pick that gives the commit already in master a different git sha1 hash?

Reply:
never mind, I think the commit is correct with the same sha1 hash as the master. the github's way of doing it confused me. Sorry about that.

Reply:
That was weird, it's fixed now though.